### PR TITLE
568 add application entity

### PR DESCRIPTION
**Pull request by rpoletaev**

### CODE REVIEW

File: packages/migration/ecosysystem.go [hunk 1]
```diff
@@ -758,12 +758,15 @@ var (
...
+		DROP TABLE IF EXISTS "%[1]d_menu"; 
+		CREATE TABLE "%[1]d_menu" (
 			"id" bigint  NOT NULL DEFAULT '0',
 			"name" character varying(255) UNIQUE NOT NULL DEFAULT '',
 			"title" character varying(255) NOT NULL DEFAULT '',
 			"value" text NOT NULL DEFAULT '',
-			"conditions" text NOT NULL DEFAULT ''
+			"conditions" text NOT NULL DEFAULT '',
+			"app_id" bigint NOT NULL DEFAULT '0',
+			"app_deleted" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: unnecessary column, if we need apps deleted, we will use joins

File: packages/migration/ecosysystem.go [hunk 2]
```diff
@@ -817,7 +820,9 @@ If("#key_id#" == EcosysParam("founder_account")){
 			"value" text NOT NULL DEFAULT '',
 			"menu" character varying(255) NOT NULL DEFAULT '',
 			"validate_count" bigint NOT NULL DEFAULT '1',
-			"conditions" text NOT NULL DEFAULT ''
+			"conditions" text NOT NULL DEFAULT '',
+			"app_id" bigint NOT NULL DEFAULT '0',
+			"app_deleted" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: same

File: packages/migration/ecosysystem.go [hunk 3]
```diff
@@ -831,7 +836,9 @@ If("#key_id#" == EcosysParam("founder_account")){
 			"id" bigint  NOT NULL DEFAULT '0',
 			"name" character varying(255) UNIQUE NOT NULL DEFAULT '',
 			"value" text NOT NULL DEFAULT '',
-			"conditions" text NOT NULL DEFAULT ''
+			"conditions" text NOT NULL DEFAULT '',
+			"app_id" bigint NOT NULL DEFAULT '0',
+			"app_deleted" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: same

File: packages/migration/ecosysystem.go [hunk 4]
```diff
@@ -850,7 +857,9 @@ If("#key_id#" == EcosysParam("founder_account")){
 		"wallet_id" bigint NOT NULL DEFAULT '0',
 		"token_id" bigint NOT NULL DEFAULT '1',
 		"active" character(1) NOT NULL DEFAULT '0',
-		"conditions" text  NOT NULL DEFAULT ''
+		"conditions" text  NOT NULL DEFAULT '',
+		"app_id" bigint NOT NULL DEFAULT '0',
+		"app_deleted" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: same

File: packages/migration/ecosysystem.go [hunk 5]
```diff
@@ -899,7 +908,9 @@ If("#key_id#" == EcosysParam("founder_account")){
 		"name" varchar(100) UNIQUE NOT NULL DEFAULT '',
 		"permissions" jsonb,
 		"columns" jsonb,
-		"conditions" text  NOT NULL DEFAULT ''
+		"conditions" text  NOT NULL DEFAULT '',
+		"app_id" bigint NOT NULL DEFAULT '0',
+		"app_deleted" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: same

**Review by potehinre — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by potehinre — APPROVED** [review]

**Pull request merged**